classes/form/lockcertificate.php

```diff
@@ -53,6 +53,19 @@ protected function definition() {
         $html .= html_writer::end_div();
 
         $mform->addElement('html', $html);
+
+        $html  = html_writer::start_div('warning');
```

Comment:
.warning isn't a css class, curious why it is here at all?

Reply:
@brendanheywood  this is done in a couple other places that I can see, 
https://github.com/catalyst/moodle-tool_lockstats/blob/master/classes/form/reset.php#L55
https://github.com/catalyst/moodle-tool_crawler/blob/master/classes/form/reset.php#L62

As well as in this file for the other warning message

Reply:
It makes the text bold